tangelo/linq/tests/test_simulator.py

```diff
@@ -368,6 +368,37 @@ def test_get_exp_value_from_statevector_h2(self):
         if test_fail:
             assert False
 
+    def test_get_exp_value_h2_with_n_shots(self):
+        """ Get expectation value of large circuits and qubit Hamiltonians corresponding to molecules.
+            Molecule: H2 sto-3g = [("H", (0., 0., 0.)), ("H", (0., 0., 0.741377))] with increasing shots
+            obtains better accuracy but never the exact result.
+        """
+        qubit_operator = load_operator("mol_H2_qubitham.data", data_directory=path_data, plain_text=True)
+
+        with open(f"{path_data}/H2_UCCSD.qasm", "r") as circ_handle:
+            openqasm_circ = circ_handle.read()
+
+        abs_circ = translate_c(openqasm_circ, "tangelo", source="openqasm")
+        expected = -1.1372704
+        test_fail = False
+
+        for n_shots in [10**2, 10**4, 10**6]:
```

Comment:
Let's use a variable `shot_counts = [10**2, 10**4, 10**6]`, that will help with the assert statement I suggest below.